rust_snuba/src/processors/snapshots/rust_snuba__processors__tests__schemas@events-errors-events__1__null-tag-keys.json.snap

```diff
@@ -0,0 +1,62 @@
+---
```

Comment:
I'm guessing this changed because there is some other schema change bundled into that schema bump?

Reply:
Yeah, honestly I am not really sure what this cargo insta thing is doing. It's really confusing